virt-spawn

```diff
@@ -105,6 +105,7 @@ Example:
 USAGE
 }
 
+ACTION=build
```

Comment:
Using a single ACTION variable instead of one per action to hopefully keep it tidier if more actions are added.